ci/docker/linux_opensuse_tumbleweed

```diff
@@ -0,0 +1,19 @@
+FROM opensuse/tumbleweed
+RUN \
+  zypper -n update && \
+  zypper -n install \
+  shadow util-linux util-linux-systemd systemd-devel \
```

Comment:
This seems to have failed quite a few times: https://cirrus-ci.com/task/5207362771353600 
Do you have any idea why that is? Perhaps there's a better mirror?

Reply:
I am not sure about that, it didn't give errors in any other run. I added `zypper ref` for refreshing mirrors, just in case.